content/languages/python/authoring/index.md

```diff
@@ -0,0 +1,255 @@
+---
+kind: tutorial
+languages: [python]
+sidebar: "language:python"
+---
+
+# Python: creating and translating a kata
+
+This article is meant as help for kata authors and translators who would like to create new content in Python. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), shows the most common pitfalls and how to avoid them.
+
+This article is not a standalone tutorial on creating kata or translations. It's meant to be a complementary, Python-specific part of a more general set of HOWTOs and guidelines related to [content authoring](/authoring/). If you are going to create a Python translation, or a new Python kata from scratch, please make yourself familiar with the aforementioned documents related to authoring in general first. 
+
+
+## General info
+
+Any technical information related to the Python setup on Codewars can be found on the [Python reference](/languages/python/) page (language versions, available modules, and setup of the code runner).
+
+
+## Description
+
+Python code blocks can be inserted with Python-specific part in [sequential code blocks](/references/markdown/extensions/#sequential-code-blocks):
+
+~~~
+```python
+
+...your code here...
+
+```
+~~~
+
+Python-specific paragraphs can be inserted with [language conditional rendering](/references/markdown/extensions/#conditional-rendering):
+
+```
+~~~if:python
+
+...text visible only for Python description...
+
+~~~
+
+~~~if-not:python
+
+...text not visible in Python description...
+
+~~~
+```
+
+## Tasks and Requirements
+
+Some concepts don't always translate well to or from Python. Because of this, some constructs should be avoided and some translations just shouldn't be done.
+- Avoid returning different data types depending on the situation (_"Return the result, or the string 'Error' if no result can be found..."_). Python is dynamically typed which is not be the case for some other languages. Returning `None` might be appropriate in some situations, but raising an exception might be better in others.
+
+Some kata just should not be translated into Python because it can be difficult to keep their initial idea:
+- The Python standard library is very rich and has many utilities available (e.g. `itertools`, combinatorics functions), so some nontrivial requirements in other languages could become trivial in Python,
+- Python supports big integers natively, so kata that rely on the implementation of arbitrary precision integer arithmetic would become trivial in Python.
+- The Codewars runner provides a set of preinstalled packages which are available for users solving the kata. They can be a real game-changer when it comes to the difficulty of a Python translation. For example, `numpy` can make many matrix manipulation kata much easier.
```

Comment:
What about a link to https://docs.codewars.com/languages/python/ since it has the list?

Reply:
It's mentioned at the beginning, you think it's worth repeating here?

Reply:
ah, didn't see. It would be nice if that was included in the sidebar here tho:
![image](https://user-images.githubusercontent.com/40404519/103142802-b7c13800-46cf-11eb-8cd8-93951045c24b.png)

Reply:
Sidebar here is built automatically, i am not sure if it can be modified. I will think about something.

Reply:
It doesn't show all the links though, apparently only references.

Reply:
Actually, it can be changed in the `sidebar` property in the gridsome.config.js.

Reply:
See #195